packager/live_packager_export.cc

```diff
@@ -105,8 +91,7 @@ LivePackagerStatus_t livepackager_package(LivePackager_t lp,
   shaka::Status status =
       lp->inner->Package(input_init, input_media, *dest->inner);
   std::string status_message = status.ToString();
-  return new (std::nothrow)
-      LivePackagerStatus_s{status_message.c_str(), status.ok()};
+  return LivePackagerStatus_s{strdup(status_message.c_str()), status.ok()};
```

Comment:
We probably don't want to copy the string if there's no error  (`strdup` will still allocate memory for the null terminator).  So we could do something like:

```
return LivePackagerStatus_t{
    status.ok() ? NULL : strdup(status_message.c_str()), 
    status.ok()
  };
```

This will make the memory management easier in CGO too.